src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
According to the checklist, it's recommended to have a single `preconnect` link to Google Fonts. You can remove the second `preconnect` to `fonts.gstatic.com` since it's not necessary when you have the first one to `fonts.googleapis.com`.